chemprop/data/datapoints.py

```diff
@@ -11,7 +11,7 @@
 MoleculeFeaturizer = Featurizer[Chem.Mol, np.ndarray]
 
 
-@dataclass(slots=True)
+@dataclass(slots=True, kw_only=True)
```

Comment:
This allows for removing `_MoleculeDatapointMixin`.

Reply:
it also is not backwards compatible